src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalPrice = days * 40;
+
+  if (days >= 7) {
+    return totalPrice - 50;
+  }
+
+  if (days >= 3) {
+    return totalPrice - 20;
+  } else {
+    return totalPrice;
+  }
```

Comment:
you don't need else here, you have return statements in both blocks
```suggestion
  }
  
  return totalPrice;
```